src/css/components/Popup.scss

```diff
@@ -1,3 +1,6 @@
+$Popup-bg-color: $cu-foreground;
```

Comment:
I like having these settings up here, but kinda think unless they're set to `!default` it might just be an extra step.  The value of these variables is to provide people with the ability to make changes through a settings file rather than the CSS itself.  When we discuss theming let's circle back on these element variables

Reply:
Oh, right - I still see them being set on a settings file level. I can add `!default` to these.